src/components/SignOutButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { withFirebase } from "./Firebase";
+import Button from '@material-ui/core/Button';
+
+const SignOutButton = ({ firebase }) => (
+  <Button
+      variant="contained"
+      color="primary"
+      onClick={() => firebase.doSignOut()}
```

Comment:
You can just pass function reference here
```suggestion
      onClick={firebase.doSignOut}
```

Reply:
good catch